internal/build/docker_test.go

```diff
@@ -132,11 +119,22 @@ func (f *testFixture) newBuilderForTesting() *localDockerBuilder {
 	return NewLocalDockerBuilder(f.dcli)
 }
 
-func (f *testFixture) assertFileInImageWithContents(tag digest.Digest, path string, contents string) {
+type pathContent struct {
+	path     string
+	contents string
+}
+
+func (f *testFixture) assertFilesInImageWithContents(tag digest.Digest, contents []pathContent) {
 	ctx := context.Background()
+	var cmd strings.Builder
+	for _, c := range contents {
+		cs := fmt.Sprintf("cat %s | grep \"%s\" || echo \"contents not found\"; ", c.path, c.contents)
```

Comment:
hmm do we want to differentiate the error message between file not found and contents not found? (at very least, err should read `file not found or has wrong contents`). or can you maybe printf the file name or the bad contents into the `echo`? this doesn't give the user much debug info...

Reply:
I couldn't think of a way to do this, but in my experience getting the stdout printed to you is enough debug info.

PRs welcome! :)

Reply:
` echo \"could not find file %s / did not have contents %s\"` and sprintf it in